tests/integration/cards.test.js

```diff
@@ -525,6 +525,121 @@ describe('CARDS ROUTES - PUT /cards/{_id}/answer/{answerId}', () => {
   });
 });
 
+describe('CARDS ROUTES - DELETE /cards/{_id}/answers/{answerId}', () => {
+  let authToken = null;
+  beforeEach(populateDB);
+  const card = cardsList[11];
+  const answer = card.questionAnswers[0];
+  const params = { _id: card._id, answerId: answer._id };
+
+  describe('VENDOR_ADMIN', () => {
+    beforeEach(async () => {
+      authToken = await getToken('vendor_admin');
+    });
+
+    it('should delete an answer', async () => {
+      const response = await app.inject({
+        method: 'DELETE',
+        url: `/cards/${card._id.toHexString()}/answers/${answer._id.toHexString()}`,
+        payload: { params },
+        headers: { 'x-access-token': authToken },
+      });
+
+      expect(response.statusCode).toBe(200);
+
+      const cardUpdated = await Card.findById(card._id).lean();
+      expect(cardUpdated).toEqual(expect.objectContaining({
+        ...card,
+        questionAnswers: [
+          card.questionAnswers[1],
+        ],
```

Comment:
Ce test ne devrait pas passer j'ai l'impression, normalement on n'autorise pas la suppresion d'une proposition si il y en a deux ou moins. (ce qui est le cas ici, non ?)

Reply:
En réalité le test du preHandler des deux questionAnswers ou moins était cassé. C'est réparé dans le commit 'Fix review and add missing constant'